hydromt_wflow/wflow.py

```diff
@@ -2576,6 +2576,7 @@ def setup_lulcmaps_with_paddy(
                 logger=self.logger,
             )
             self.set_grid(soil_maps["kvfrac"], name="kvfrac")
+            self.set_config("vertical.kvfrac", "kvfrac")
```

Comment:
My bad you still need to specify input for the toml file before the variable name
```suggestion
            self.set_config("input.vertical.kvfrac", "kvfrac")
```